domain/BUILD.bazel

```diff
@@ -134,13 +134,16 @@ go_test(
         "//parser/ast",
         "//parser/auth",
         "//parser/model",
+        "//parser/mysql",
         "//parser/terror",
         "//server",
         "//session",
+        "//sessionctx/stmtctx",
         "//sessionctx/variable",
         "//store/mockstore",
         "//testkit",
         "//testkit/testsetup",
+        "//types",
```

Comment:
why?

Reply:
It is the bazel config. it need to update the dependency to speed up build/linter/UT.